skins/laika/templates/Base_Layout.html.twig

```diff
@@ -28,7 +28,8 @@
 </head>
 <body>
 {% block main %}
-    <div id="app" data-application-vars="{$ _context|json_encode|e('html_attr') $}" 
+    <div id="app" />
```

Comment:
This element needs a closing tag, otherwise the browser thinks the appdata one is a child and it gets removed when the app gets mounted.